1.1/SparseImplicitQuadtree/README.md

```diff
@@ -0,0 +1,34 @@
+
+# Sparse implicit quadtree
+
+An example tileset that uses the [Implicit Tiling](https://github.com/CesiumGS/3d-tiles/tree/draft-1.1/specification/ImplicitTiling) to represent a small, sparse quadtree. 
+
+The quadtree has 6 available levels, and each subtree has 3 levels. There are 32 tiles available in level 5. Each of these tiles has a content, which is a simple glTF asset as a GLB (glTF binary) file that just consists of a portion of the unit square that corresponds to the extent of the respective tile. No other tiles - except for the ones that have content, and their respective ancestors - are available. 
+
+A screenshot of the tileset, rendered in CesiumJS (with a bounding volume visualization for debugging) is shown here:
+
+![Screenshot](screenshot/SparseImplicitQuadtree.png)
+
+It was created from a Cesium Sandcastle with the following code:
+```JavaScript
+var viewer = new Cesium.Viewer('cesiumContainer');
+
+var tileset = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
+    url : 'http://localhost:8003/1.1/SparseImplicitQuadtree/tileset.json',
+    maximumScreenSpaceError: 1,
```

Comment:
Solved via https://github.com/javagl/3d-tiles-samples/commit/b1e8890f9918ae0fc7a9c9e2719b8153ea8ddcb3

The screen space error is now the default one, and the tileset has been scaled up to 15 meters. (Only in the tileset model matrix. I'd _really_ like the actual geometry data to cover the unit square, because ... ... ... I'd really like it to be this way).